contracts/dynamic-caller-contract/src/contract.rs

```diff
@@ -24,6 +24,8 @@ struct CalleeContract {
     address: Addr,
 }
 
+// When `user_defined_mock = true`, user can and must write mock implement
```

Comment:
How about adding "mock implement _for test_"? Is it verbose?

Reply:
Thank you. Sure, it's better to specify that it's a test. I fixed it like that.